.github/workflows/deploy-sync-check.yml

```diff
@@ -30,3 +30,4 @@ jobs:
           slack_token: ${{ secrets.SLACK_TOKEN }}
           working_directory: sync_check_terraform/
           environment: Sync Check Service
+          NR_LICENSE_KEY: ${{ secrets.NR_LICENSE_KEY }}
```

Comment:
nit: need to add a new line at EOF. Would recommend installing a linter going forward, easier to fix and verify.